requirements.txt

```diff
@@ -9,6 +9,7 @@ google-api-python-client==1.5.5
 httplib2==0.9.2
 oauth2client==4.0.0
 oauthlib==2.0.0
+psycopg2==2.6.2
```

Comment:
I don't see any changes to the code above that adds in this dependency, where did it come from?